docs/usage/private-modules.md

```diff
@@ -1,28 +1,37 @@
+description: How to support private npm modules when using Renovate
 ---
 
 # Private npm module support
 
 ## When are npm private modules used?
 
-Private npm modules are used at two times during Renovate's process.
+Private npm modules are used at two times during Renovate's process:
 
-#### 1. Module lookup
+1. Module lookup
+1. Lock file generation
 
-If a private npm module is listed as a dependency in a `package.json`, then Renovate will attempt to keep it up-to-date by querying the npm registry like it would for any other package.
-Hence, by default with no configuration a private package lookup will fail, because of lack of credentials.
-This means it won't be "renovated" and its version will remain unchanged in the package file unless you update it manually.
-These failures don't affect Renovate's ability to look up _other_ modules in the same package file.
+This will be explained below.
 
-Assuming the private module lookup succeeds (solutions for that are described later in this document) then private package versions will be kept up-to-date like public package versions are.
+### Module lookup
 
-#### 2. Lock file generation
+If a private npm module is listed as a dependency in the `package.json` file Renovate attempts to keep it up-to-date.
+Renovate queries the npm registry like it does for any other package.
+By default Renovate runs without any credentials, and so any private package lookup will fail.
+This means, that any private package will not be renovated, and must be updated manually.
+The failure to authenticate doesn't affect Renovate's ability to look up _other_ modules in the same package file.
 
-If you are using a lock file (e.g. Yarn's `yarn.lock` or npm's `package-lock.json`) then Renovate needs to update that lock file whenever _any_ package listed in your package file is updated to a new version.
+You can fix the private package lookup problem by providing the Renovate bot with the proper credentials.
+This is described later in this document.
 
-To do this, Renovate will run `npm install` or equivalent and save the resulting lock file.
+Assuming the private module lookup succeeds then private package versions will be kept up-to-date like the public package versions are.
+
+### Lock file generation
+
+If you use a lock file (e.g. Yarn's `yarn.lock` or npm's `package-lock.json`) then Renovate must update that lock file whenever _any_ package listed in your package file is updated to a new version.
+
+To do this, Renovate runs `npm install` or equivalent and saves the resulting lock file.
 If a private module hasn't been updated, it _usually_ won't matter to npm/Yarn because they won't attempt to update its lock file entry anyway.
 However it's possible that the install will fail if it attempts to look up that private module for some reason, even when that private module is not the main one being updated.
 It's therefore better to provide Renovate with all the credentials it needs to look up private packages.
```

Comment:
This is really hard to understand for me.
I would rewrite this myself but I don't really understand the text we have now, so I'll leave this for the maintainers to fix. 😄